lib/tasks/payment_breakdown.rake

```diff
@@ -28,23 +30,36 @@ namespace :payment_calculation do
       event_type: :started,
     )
 
-    service_fee = number_to_delimited(breakdown.dig(:service_fees, :service_fee_monthly).to_i)
+    service_fees = breakdown.dig(:service_fees).each_with_object([]) do |hash, bands|
+      bands << [
+        band_name_from_index(bands.length),
+        "£#{number_to_delimited(hash[:service_fee_monthly].to_i)}",
+        "£#{number_to_delimited(hash[:service_fee_per_participant].to_i)}",
+      ]
+    end
     output_payment = number_to_delimited(breakdown.dig(:output_payment, :started, :subtotal).to_i)
 
-    output = <<-RESULT
-      ---------------------------------------------
-      |  Payment type            | Payment amount |
-      ---------------------------------------------
-      | Service fee (monthly)    |   £#{service_fee}      |
-      | Output payment (started) |   £#{output_payment}     |
-      ---------------------------------------------
-      Based on #{number_to_delimited(total_participants.to_i)} participants and £#{per_participant} per participant
+    table = Terminal::Table.new(
+      title: "Breakdown Payments",
+      headings: ["Banding", "Service fee (PP)", "Service fee (monthly)"],
+      rows: service_fees,
+    )
+    table.style = { alignment: :center }
+
+    output = <<~RESULT
+      Output payment (started) £#{output_payment}
+      Based on #{number_to_delimited(total_participants.to_i)} participants and #{per_participant_in_bands}
     RESULT
 
+    logger.info table
```

Comment:
Alright, that is getting a little silly. If we want it to look presentable, maybe let's use html? I know we went for a minimum effort console approach first, but adding a new gem that lets you style things in console really doesn't sound that minimal for me. @jamesrose thoughts? 

Maybe two-step controller, step one being a simple form for inputting what you want and step two a view displaying something comparable to this? We can only have this route in non-real environments if it makes our lives easier...

Reply:
Could use html and 2-step controller of course

Reply:
@kicferk1 This is all throw-away for now. As long as this task is just _formatting_ data returned by `PaymentCalculator::Ecf::PaymentCalculation` then I'm content. I'm keen to hold off on a UI until we know how it would be used.

Saying that, if there are things here that you wouldn't want to delete _today_ I would look to move them into the calculator. One day very soon this file won't exist.